I am a PhD candidate in bioinformatics, a first class honours graduate in computer engineering, a computer and electronics enthusiast, an open-source software developer and advocate, an avid traveller, a hiker, a singer, and an experimental cook. I have sixteen years of industry and academic experience developing software in C++, Python, and R, including ten years developing high-performance distributed and parallel algorithms for genome sequence assembly, and six years developing embedded and real-time control systems. I have extensive experience developing portable software for POSIX systems such as Linux and macOS. When developing performance-critical software, I understand the hardware from the transistor up.

## Education

#### 2012–2019—PhD Bioinformatics—University of British Columbia

NSERC Canada Graduate Scholarship (CGS) student in bioinformatics. Coursework in bioinformatics, statistics, machine learning, and genetics. GPA 4.26 (93%)

#### 1998–2004—BASc Computer Engineering—Simon Fraser University

First class honours scholarship student in computer engineering. Completed an honours undergraduate thesis and two years of co-op work experience at three companies. GPA 3.85

## Work Experience

#### 2008–2012—Computational biologist—BC Cancer Genome Sciences Centre

Developed the genome sequence assembly software package, ABySS, implemented in C++. This heavily parallel and distributed software system, which uses both MPI and OpenMP for parallel computation, was the first system to assemble a human genome using short-read sequencing technology.

#### 2004–2007—Embedded firmware developer—Pathway Connectivity

Developed the firmware for Pathway's product line of lighting control devices. These devices, which use Atmel's AVR and ARM microcontrollers, convert between a variety of lighting-control protocols. Modernized the lighting control industry in developing a device, the Pathport, to adapt legacy serial equipment to an Ethernet network. This device won an award for best new product at the industry trade show, Lighting Dimensions International.

## Volunteer Experience

#### 2002–present—Open-source developer

Developer and leadership committee member of Homebrew, the package manager for macOS and Linux. Leader maintainer and creator of Homebrew for Linux, which has over fifteen thousand users.

## Expertise

#### Bioinformatics and genomics

Techniques for analysis of whole-genome shotgun sequencing and RNA-seq, including genome sequence assembly, sequence alignment, variant calling, genome annotation, and visualization and inspection of genome assemblies, variants, and rearrangements.

#### Programming languages

C++, C, Python, R, Ruby, shell, and assembly.

#### Parallel processing

OpenMP, MPI, and POSIX threads.

## Selected Publications

A selection from over 30 peer-reviewed publications, h-index of 21, and over 8,000 citations

#### ORCA: A Comprehensive Bioinformatics Container Environment for Education and Research
**SD Jackman**, T Mozgacheva, S Chen, *et al.*
(2019)
*Bioinformatics*

#### Tigmint: Correcting assembly errors using linked reads from large molecules
**SD Jackman**, L Coombe, J Chu, *et al.*
(2018)
*BMC Bioinformatics*

#### ABySS 2.0: Resource-efficient assembly of large genomes using a Bloom filter
**SD Jackman**, BP Vandervalk, H Mohamadi, J Chu, S Yeo, SA HammondA, G Jahesh, H Khan, L Coombe, RL Warren, *et al.*
(2017)
*Genome Research*

#### Organellar Genomes of White Spruce (*Picea glauca*): Assembly and Annotation
**SD Jackman**, RL Warren, EA Gibb, BP Vandervalk, H Mohamadi, J Chu, A Raymond, S Pleasance, R Coope, MR Wildung, *et al.*
(2015)
*Genome Biology and Evolution*

#### Improved white spruce (*Picea glauca*) genome assemblies and annotation of large gene families of conifer terpenoid and phenolic defense metabolism
RL Warren, CI Keeling, MMS Yuen, A Raymond, GA Taylor, BP Vandervalk, H Mohamadi, D Paulino, R Chiu, **SD Jackman**, *et al.*
(2015)
*The Plant Journal*

#### UniqTag: Content-Derived Unique and Stable Identifiers for Gene Annotation
**SD Jackman**, J Bohlmann, I Birol.
(2015)
*PLOS ONE*

#### Assembling the 20 Gb white spruce (*Picea glauca*) genome from WGSS data
I Birol, A Raymond, **SD Jackman**, *et al.*
(2013)
_Bioinformatics_

#### Draft genome of the mountain pine beetle, *Dendroctonus ponderosae* Hopkins
CI Keeling, MM Yuen, NY Liao, TR Docking, SK Chan, GA Taylor, DL Palmquist, **SD Jackman**, *et al.*
(2013)
_Genome Biology_

#### De novo assembly and analysis of RNA-seq data
G Robertson, J Schein, R Chiu, R Corbett, M Field, **SD Jackman** *et al.*
(2010)
_Nature Methods_

#### Assembling genomes using short-read sequencing technology
**SD Jackman**, I Birol
(2010)
_Genome Biology_

#### De novo transcriptome assembly with ABySS
I Birol, **SD Jackman**, CB Nielsen *et al.*
(2009)
_Bioinformatics_

#### ABySS-Explorer: visualizing genome sequence assemblies
CB Nielsen, **SD Jackman**, I Birol *et al.*
(2009)
_IEEE Transactions on Visualization and Computer Graphics_

#### ABySS: a parallel assembler for short read sequence data
JT Simpson, K Wong, **SD Jackman** *et al.*
(2009)
_Genome Research_
